Source/WebKit/UIProcess/PageClient.h

```diff
@@ -25,6 +25,7 @@
 
 #pragma once
 
+#include "ColorControlSupportsAlpha.h"
```

Comment:
Nit - you could consider forward declaring this enum instead of importing the header:

```
namespace WebKit {
enum class ColorControlSupportsAlpha : bool;
}
```

...and then you only need to pull in header if you're using one of the concrete values.